run.py

```diff
@@ -4,7 +4,7 @@
     import sys
 
     if sys.version_info[0:2] < (3, 6):
```

Comment:
` if sys.version_info[0:2] < (3, 8):`

Reply:
Yep, missed that for some reason, sorted now.